common/views/components/ImageViewer/ImageViewer2.js

```diff
@@ -0,0 +1,269 @@
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.escapeCloseViewer);
+  }
+
+  render() {
+    return (
+      <div className={`${this.props.classes} image-viewer__content image-viewer__content2`}>
+        <Script
+          url='https://static.wellcomecollection.org/openseadragon/openseadragon.min.js'
+          onError={this.handleScriptError}
+          onLoad={this.handleScriptLoaded}
+        />
+        <div className="image-viewer__controls flex flex-end flex--v-center">
+          <ButtonButton
+            text='Zoom in'
+            id={`zoom-in-${this.props.id}`}
+            icon='zoomIn'
+            eventTracking={`{${commonBtnTracking(this.props.id, this.props.trackTitle)}, "action": "work-zoom-in-button:click"}`}
+            extraClasses={`${buttonFontClasses} btn--round btn--black ${spacing({s: 1}, {margin: ['right']})}`}
+          />
+
+          <ButtonButton
+            text='Zoom out'
+            id={`zoom-out-${this.props.id}`}
+            icon='zoomOut'
+            eventTracking={`{${commonBtnTracking(this.props.id, this.props.trackTitle)}, "action": "work-zoom-out-button:click"}`}
+            extraClasses={`${buttonFontClasses} btn--round btn--black ${spacing({s: 8}, {margin: ['right']})}`}
+          />
+
+          <ButtonButton
+            text='Close image viewer'
+            icon='cross'
+            eventTracking={`{${commonBtnTracking(this.props.id, this.props.trackTitle)}, "action": "work-exit-image-viewer:btnClick"}`}
+            extraClasses={`${buttonFontClasses} btn--round btn--black js-image-viewer__exit-button ${spacing({s: 2}, {margin: ['right']})}`}
+            onClick={this.props.handleViewerDisplay}
+          />
+        </div>
+
+        <div className='image-viewer__image' id={`image-viewer-${this.props.id}`}>
+          {this.state.scriptError && <Error />}
+        </div>
+      </div>
+    );
+  }
+}
+
+type ImageViewerProps = {|
+  id: string,
+  trackTitle: string,
+  contentUrl: string,
+  width: number
+|}
+
+type ImageViewerState = {|
+  showViewer: boolean,
+  mountViewButton: boolean,
+  viewButtonMounted: boolean
+|}
+
+class ImageViewer extends React.Component<ImageViewerProps, ImageViewerState> {
+  handleViewerDisplay: Function;
+  viewButtonMountedHandler: Function;
+
+  constructor(props: ImageViewerProps) {
+    super(props);
+    this.state = {
+      showViewer: false,
+      mountViewButton: false,
+      viewButtonMounted: false
+    };
+    this.handleViewerDisplay = this.handleViewerDisplay.bind(this);
+    this.viewButtonMountedHandler = this.viewButtonMountedHandler.bind(this);
+  }
+
+  viewButtonMountedHandler() {
+    this.setState(prevState => ({
+      viewButtonMounted: !prevState.viewButtonMounted
+    }));
+  }
+
+  handleViewerDisplay(e: Event) {
+    this.setState(prevState => ({
+      showViewer: !prevState.showViewer
+    }));
+    if (e && e.target.tagName === 'IMG') {
```

Comment:
Is this check necessary? i.e. aren't we only going to call this function from a click event?

Reply:
This was for tracking clicks on the image specifically, but am going to remove the tracking stuff anyway now.